### PR TITLE
PANOS - EXPANDR-5744

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-5744

## Description
A few needed improvements to the "PAN-OS - Block Destination Service" playbook.  See the above Jira last comment for more information.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , can you please take a look

File: Packs/PAN-OS/Playbooks/playbook-PAN-OS_-_Block_Destination_Service.yml [hunk 1]
```diff
@@ -1,18 +1,15 @@
 id: PAN-OS - Block Destination Service
 version: -1
-contentitemexportablefields:
-  contentitemfields: {}
 name: PAN-OS - Block Destination Service
-description: 'This playbook blocks a Destination IP and Service (TCP or UDP port)
-  by creating a rule for a specific Device Group on PAN-OS. '
+description: 'This playbook blocks a Destination IP and Service (TCP or UDP port) by creating a rule for a specific Device Group on PAN-OS. '
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/Playbooks/playbook-PAN-OS_-_Block_Destination_Service.yml:4]:
```suggestion
description: 'This playbook blocks a destination IP and service (TCP or UDP port) by creating a rule for a specific device group on PAN-OS. '
```

File: Packs/PAN-OS/Playbooks/playbook-PAN-OS_-_Block_Destination_Service.yml [hunk 2]
```diff
@@ -199,92 +205,82 @@ tasks:
...
     id: "10"
-    taskid: a222c7cc-28e4-4e82-80fc-b3f0bb8c99ba
+    taskid: 41890197-ea43-4d94-8746-6221d39b4d83
     type: playbook
     task:
-      id: a222c7cc-28e4-4e82-80fc-b3f0bb8c99ba
+      id: 41890197-ea43-4d94-8746-6221d39b4d83
       version: -1
       name: PAN-OS Commit Configuration
+      description: Commit the PAN-OS Panorama or Firewall configuration.\nIf specified as Panorama, it also pushes the Policies to the specified Device Group in the instance.
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/Playbooks/playbook-PAN-OS_-_Block_Destination_Service.yml:245]:
```suggestion
      description: Commit the PAN-OS Panorama or firewall configuration.\nIf specified as Panorama, it also pushes the policies to the specified device group in the instance.
```

File: Packs/PAN-OS/ReleaseNotes/2_1_4.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### PAN-OS - Block Destination Service
+
+- Updated the playbook to check if address object had already been created and added WhereRule, SourceZone and DestinationZone inputs.
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/ReleaseNotes/2_1_4.md:6]:
```suggestion
- Updated the playbook to check if the address object had already been created.
- Added the following inputs:
   - WhereRule
   - ,SourceZone
   - DestinationZone
```

**Comment by ShirleyDenkberg** [comment]
@melamedbn @ostolero Doc review completed.

**Comment by ostolero** [comment]
Hi @johnnywilkes , thank you for the contribution the changes look good! I've reached out through slack to schedule a demo.

**Comment by ostolero** [comment]
Hi @johnnywilkes thank you for supplying the video demo! me and @efelmandar are going over it and will return with feedback soon.

**Review by efelmandar — COMMENTED** [review]
@johnnywilkes I reviewed the changes and it looks excellent. Thank you for improving this playbook.
@ostolero approved

**Review by ostolero — APPROVED** [review]
Great work! merging

**Comment by ostolero** [comment]
Hi @johnnywilkes, as there is a conflict with our master we can't merge. What you can do is either solve the conflict yourself (it's simply the rn overlapping with other changes, shouldn't be too complicated). Or grant me permission to push to your branch, An explanation on how to do that is [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository).

**Comment by johnnywilkes** [comment]
@ostolero , fixed, please merge

**Pull request merged**